WeatherTests/UnitTests/CityValidatorTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WeatherBL.Validators;
+
+namespace WeatherTests.UnitTests
+{
+    [TestFixture]
+    public class CityValidatorTests
+    {
+        private readonly CityValidator _cityValidator;
+        private string city;
+        private string nullCity;
+
+        public CityValidatorTests()
+        {
+            _cityValidator = new CityValidator();
+        }
+
+        [Test]
+        public void Validate_CityNameIsNotNull_ReturnsIsNotNull()
+        {
+            //Arrange
+            city = "London";
+
+            //Act
+            var result = _cityValidator.Validate(city);
+
+            //Assert
+            Assert.IsNotNull(result);
```

Comment:
What is the reason to check for null? Method returns bull. Can the result be null? Think about another test case

Reply:
fixed